app/controllers/search_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SearchController < ApplicationController
+  def index
+    @food_info = FoodFacade.foods_containing(params[:q])
+    @food_count = FoodFacade.count_foods_containing(params[:q])
```

Comment:
Both of these methods, `foods_containing` and `count_foods_containing` call the same endpoint, and just pull different data. I'd like to challenge you to find a way to just call one method in your facade, and have your facade take care of combining the food objects and the totalHits into one object to get returned to your controller. All of the data that you need for this view can come from the one query call you're making.

Reply:
You could look at using syntax like this: 

`food_count , food_info = FoodFacade.your_method(params[:q])`
Where `your_method` will return a 2d array with the first element being the food count, and the second element being the food object collection.